rocAL/rocAL_pybind/amd/rocal/pipeline.py

```diff
@@ -410,12 +405,12 @@ def create_pipeline(*args, **kwargs):
             with pipe:
                 pipe_outputs = func(*args, **fn_kwargs)
                 if isinstance(pipe_outputs, tuple):
-                    po = pipe_outputs
+                    outputs = pipe_outputs
```

Comment:
why is this renamed. po (pipe_outputs) was fine

Reply:
Reverted back to the original name for the variable